docs/README.md

```diff
@@ -1,12 +1,12 @@
-<!--
----
-title: "Triggers and EventListeners"
-linkTitle: "Triggers"
-weight: 3
-description: >
-  Event Triggers
+## <!--
+
+title: "Triggers and EventListeners" linkTitle: "Triggers" weight: 3
+description: > Event Triggers
+
 ---
+
 -->
```

Comment:
Oh-oh, Prettier doesn't like the commented out front matter for the website (https://github.com/tektoncd/triggers/pull/500). Unless there's a way to add the front matter and make Prettier happy, we will have to disable these automated Prettier PRs.

cc @dibyom

Reply:
Prettier doesn't complain about the `update-docs` comments (like [this](https://github.com/tektoncd/triggers/blame/master/docs/eventlisteners.md#L80) one). So, there might be something we can do to the front matter comments to stop Prettier from adding `##`?

Reply:
Hmm yeah looks like its messes up the formatting. I assume that the website depends on it @skaegi ? 🤕

Reply:
Adding a `<!-- prettier-ignore -->` might help. 

https://prettier.io/docs/en/ignore.html#markdown

Reply:
Sorry just found this now -- yes it's for the website